libs/perun/dialogs/src/lib/password-reset-dialog/password-reset-dialog.component.html

```diff
@@ -0,0 +1,60 @@
+<ng-template #spinner>
+  <perun-web-apps-loading-dialog></perun-web-apps-loading-dialog>
+</ng-template>
+<div class="user-theme position-relative">
+  <div class="title-with-icon">
+    <h1 mat-dialog-title>
+      {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.TITLE' | translate}}
+    </h1>
+  </div>
```

Comment:
There is no icon in the dialog title.
```suggestion
  <h1 mat-dialog-title>
    {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.TITLE' | translate}}
  </h1>
```